source/d3d11/d3d11_device.cpp

```diff
@@ -64,7 +64,7 @@ HRESULT STDMETHODCALLTYPE D3D11Device::QueryInterface(REFIID riid, void **ppvObj
 		riid == __uuidof(ID3D11Device2) ||
 		riid == __uuidof(ID3D11Device3))
 	{
-		#pragma region Update to ID3D11Device1 interface
+#pragma region Update to ID3D11Device1 interface
```

Comment:
This entire file can be reverted, since it's just (probably unintended) whitespace changes.

Reply:
Made the change. I commented this line because, in debug mode, it makes Reshade crash... Do you know why it behave like that ?

Reply:
yes, it is. Whare ate your format settings for the whitespaces in your IDE ?

Reply:
I'm assuming you mean the change above. You need the DX debug layers installed for that flag to work. That should usually happen when you install the Windows SDK via Visual Studio though.